src/con_duct/__main__.py

```diff
@@ -511,10 +514,89 @@ def dump_json(self) -> str:
 
     @property
     def execution_summary_formatted(self) -> str:
-        human_readable = {
-            k: "unknown" if v is None else v for k, v in self.execution_summary.items()
-        }
-        return self.summary_format.format_map(human_readable)
+        formatter = Formatter()
+        return formatter.format(self.summary_format, **self.execution_summary)
+
+
+class Formatter(string.Formatter):
+    # TODO: we might want to just ignore and force utf8 while explicitly .encode()'ing output!
+    # unicode versions which look better but which blow during tests etc
+    # Those might be reset by the constructor
+    OK = "OK"  # u"✓"
+    NOK = "X"  # u"✗"
+    NONE = "-"  # u"✗"
+
+    def __init__(self, *args, **kwargs):
+        super(Formatter, self).__init__(*args, **kwargs)
+        if sys.stdout.encoding is None:
```

Comment:
we are printing to log which is printing to stderr... but indeed it is only distantly related.

anyways -- can remove these unicodes etc -- this all was to verify that we could use them. If you don't want to use them ,get rid of them. If you want to use them -- we might get surprised on some environment when it would start puking that it can't log them for some reason.

Reply:
+1 unicode removed, so this block was no longer need.